inference/mlflow-huggingface/src/models/bert-base-uncased/model.yaml

```diff
@@ -1,11 +1,8 @@
 path:                                                                 #Required
-  # local: "./local-path-to-model-files"                              #Required  Either remote or local should be passed. if both are given, local is given preference  
-  remote:                                                             #Required. Either remote or local should be passed. if both are given, local is given preference  
-    type: git                                                         #Required. Download Type. Can be git, http or ftp
-    uri: https://huggingface.co/bert-base-uncased                     #Required. Download URI
-    commit_hash: 5546055f03398095e385d7dc625e636cc8910bf2             #Optional
```

Comment:
and i think you probably want to capture the hash in the asset itself too, it makes it easier for you to do linear tracking

Reply:
I was thinking it will be great to capture commit hash at our end instead of publisher sharing that in config. Unless we have a requirement that we want to register model with changes upto a particular commit, and not the latest one all the time. Please let me know your thoughts as well. 

cc: @skasturi @praveenmathamsetty

Reply:
As an infrastructure, I believe adding some kind of hash is necessary when 3P developers/customers try to onboard their models and as @wayliums mentioned, tracking.

That said, `commit_hash` is relevant only in the context of `git`. 

I would suggest having multiple options: 

1. checksum (different algos) - http based downloads
2. commit_hash (git)

Reply:
.. optional though.

Reply:
tracking separately.